interfaces/vrep/DQ_VrepInterface.m

```diff
@@ -1034,7 +1020,126 @@ function set_joint_target_velocities(obj,jointnames,joint_target_velocities,opmo
                         opmode);
                 end                
             end            
-        end         
+        end
+
+        function joint_torques = get_joint_torques(obj,handles,opmode)
```

Comment:
@ffasilva please replace `handles` by` jointnames` in the function `get_joint_torques(obj,handles,opmode)`

Reply:
Done in [4f1b7c9](https://github.com/dqrobotics/matlab/pull/104/commits/4f1b7c9fa018ae6f6f93b41057182400ebffc8bb).